src/main/java/frc/lib/team3061/drivetrain/Drivetrain.java

```diff
@@ -666,10 +666,157 @@ public boolean isMoveToPoseEnabled() {
+                          }
+                        }))),
+
+        // Tests for driving to the right
+        Commands.parallel(
+            Commands.run(
+                () -> {
+                  io.driveRobotRelative(1, 0, 0, false);
+                }),
+            Commands.waitSeconds(1)
+                .andThen(
+                    Commands.runOnce(
+                        () -> {
+                          for (int i = 0; i < 4; i++) {
+                            if (inputs.swerve[i].driveVelocityMetersPerSec < 0.5) {
+                              FaultReporter.getInstance()
+                                  .addFault(
+                                      SUBSYSTEM_NAME,
+                                      "[System Check] Swerve module "
+                                          + getSwerveLocation(i)
+                                          + " not moving as fast right as expected");
+                            }
+                          }
+                        }))),
+
+        // Tests for driving wheels forwards
+        Commands.parallel(
+            Commands.run(
+                () -> {
+                  io.driveRobotRelative(0, 1, 0, false);
+                }),
+            Commands.waitSeconds(1)
+                .andThen(
+                    Commands.runOnce(
+                        () -> {
+                          for (int i = 0; i < 4; i++) {
+                            if (inputs.swerve[i].driveVelocityMetersPerSec < 0.5) {
+                              FaultReporter.getInstance()
+                                  .addFault(
+                                      SUBSYSTEM_NAME,
+                                      "[System Check] Swerve module "
+                                          + getSwerveLocation(i)
+                                          + " not moving as fast forwards as expected");
+                            }
+                          }
+                        }))),
+
+        // Tests for driving wheels backwards
+        Commands.parallel(
+            Commands.run(
+                () -> {
+                  io.driveRobotRelative(0, -1, 0, false);
+                }),
+            Commands.waitSeconds(1)
+                .andThen(
+                    Commands.runOnce(
+                        () -> {
+                          for (int i = 0; i < 4; i++) {
+                            if (inputs.swerve[i].driveVelocityMetersPerSec < 0.5) {
+                              FaultReporter.getInstance()
+                                  .addFault(
+                                      SUBSYSTEM_NAME,
+                                      "[System Check] Swerve module "
+                                          + getSwerveLocation(i)
+                                          + " not moving as fast backwards as expected");
+                            }
+                          }
+                        }))),
+
+        // Tests for driving wheels clockwise
```

Comment:
Check the above referenced coordinate system document for rotation as well. (Positive rotation is CCW, like in Physics.)